pkg/operator/starter.go

```diff
@@ -153,7 +154,20 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 			"v3.11.0/openshift-controller-manager/leader-role.yaml",
 			"v3.11.0/openshift-controller-manager/leader-rolebinding.yaml",
 			"v3.11.0/openshift-controller-manager/ns.yaml",
+			"v3.11.0/openshift-controller-manager/route-controller-informer-clusterrole.yaml",
+			"v3.11.0/openshift-controller-manager/route-controller-informer-clusterrolebinding.yaml",
+			"v3.11.0/openshift-controller-manager/route-controller-leader-role.yaml",
+			"v3.11.0/openshift-controller-manager/route-controller-leader-rolebinding.yaml",
 			"v3.11.0/openshift-controller-manager/route-controller-ns.yaml",
+			"v3.11.0/openshift-controller-manager/route-controller-sa.yaml",
+			"v3.11.0/openshift-controller-manager/route-controller-separate-sa-role.yaml",
+			"v3.11.0/openshift-controller-manager/route-controller-separate-sa-rolebinding.yaml",
+			"v3.11.0/openshift-controller-manager/route-controller-servicemonitor-role.yaml",
+			"v3.11.0/openshift-controller-manager/route-controller-servicemonitor-rolebinding.yaml",
```

Comment:
Both service monitor role and rolebinding are defined in `manifests/0000_90_openshift-controller-manager-operator_04_operand-route-controller-servicemonitor.yaml`. Will not CVO manage both role and rolebinding as well?

Reply:
Checking https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/master/manifests/0000_90_openshift-controller-manager-operator_03_operand-servicemonitor.yaml:
```
apiVersion: rbac.authorization.k8s.io/v1
kind: Role
metadata:
  name: prometheus-k8s
  namespace: openshift-controller-manager
  annotations:
    include.release.openshift.io/self-managed-high-availability: "true"
    include.release.openshift.io/single-node-developer: "true"
...
```
vs. https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/master/bindata/v3.11.0/openshift-controller-manager/servicemonitor-role.yaml:
```
apiVersion: rbac.authorization.k8s.io/v1
kind: Role
metadata:
  name: prometheus-k8s
  namespace: openshift-controller-manager
```

The former is missing the annotations. We need to revisit the manifests later to see whether we can remove the latter to remove duplicates.

Reply:
ack, let's revisit this later